ansible/roles/bastion-lite/tasks/main.yml

```diff
@@ -64,16 +61,20 @@
     owner: root
     group: root
 
-- name: Install stern (multi-pod logging tool) on the bastion
-  get_url:
-    url: https://gpte-public.s3.amazonaws.com/stern_linux_amd64
-    dest: /usr/bin/stern
-    mode: 0775
-    owner: root
-    group: root
-
-- name: Add GUID to /etc/skel/.bashrc
+- name: Add GUID to /etc/skel/.bashrc and /home/{{ ansible_user }}/.bashrc
   lineinfile:
-    path: "/etc/skel/.bashrc"
+    path: "{{ item }}"
     regexp: "^export GUID"
     line: "export GUID={{ guid }}"
+  loop:
+  - "/etc/skel/.bashrc"
+  - "/home/{{ ansible_user }}/.bashrc"
+
+- name: Add CLOUD_USER to /etc/skel/.bashrc and /home/{{ ansible_user }}/.bashrc
```

Comment:
Small nit...update the var in the name of the task to match the var you are using (no _)

Reply:
Updated